master/internal/api_experiment_intg_test.go

```diff
@@ -292,6 +292,139 @@ func TestMoveExperimentRunID(t *testing.T) {
 	})
 }
 
+func TestMoveExperiments(t *testing.T) {
+	api, curUser, ctx := setupAPITest(t, nil)
+	_, projectID := createProjectAndWorkspace(ctx, t, api)
+
+	t.Run("Move an experiement without filters", func(t *testing.T) {
+		exp := createTestExp(t, api, curUser)
+		result, err := api.MoveExperiments(ctx, &apiv1.MoveExperimentsRequest{
+			ExperimentIds:        []int32{int32(exp.ID)},
+			DestinationProjectId: int32(projectID),
+			Filters:              nil,
+		})
+		for _, v := range result.Results {
+			require.Equal(t, v.Error, "")
+		}
+		require.Equal(t, len(result.Results), 1)
+		require.NoError(t, err)
+	})
+
+	t.Run("Move multiple experiements without filters", func(t *testing.T) {
+		exp1 := createTestExp(t, api, curUser)
+		exp2 := createTestExp(t, api, curUser)
+		result, err := api.MoveExperiments(ctx, &apiv1.MoveExperimentsRequest{
+			ExperimentIds:        []int32{int32(exp1.ID), int32(exp2.ID)},
+			DestinationProjectId: int32(projectID),
+			Filters:              nil,
+		})
+		for _, v := range result.Results {
+			require.Equal(t, v.Error, "")
+		}
+		require.Equal(t, len(result.Results), 2)
+		require.NoError(t, err)
+	})
+
+	t.Run("Move experiements with filters", func(t *testing.T) {
```

Comment:
nit: I believe that there is a typo on experiments

Reply:
fixed all typos